start-utils

```diff
@@ -37,6 +37,10 @@ function isDebugging {
 
 function debug {
   if isDebugging; then
-    echo "DEBUG: $*"
+    log "DEBUG: $*"
   fi
 }
+
+function log {
+  echo "[wrapper] $*"
```

Comment:
I'm good with hardcoding this; however, "wrapper" still doesn't feel right to me. How about "[startup]" or "[init]"?

Reply:
`[init]` sound good in my ears. Even better would be to perhaps have something more descriptive. I can't figure out a good name though, since `[minecraft-server]` would indicate the actual Minecraft server.